custom_components/volkswagen_we_connect_id/config_flow.py

```diff
@@ -6,6 +6,7 @@
 
 import voluptuous as vol
 from weconnect import weconnect
+from weconnect.errors import AuthentificationError
```

Comment:
That spelling is not a typo of mine! It is how it reads in the library:
https://github.com/tillsteinbach/WeConnect-python/blob/v0.59.5/weconnect/errors.py#L17